src/org/usfirst/frc4904/sovereignty/AlignmentSystem.java

```diff
@@ -0,0 +1,58 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import org.usfirst.frc4904.robot.vision.AligningCamera;
+import org.usfirst.frc4904.standard.LogKitten;
+import org.usfirst.frc4904.standard.custom.sensors.InvalidSensorException;
+import edu.wpi.first.wpilibj.PIDSource;
+import edu.wpi.first.wpilibj.PIDSourceType;
+
+public class AlignmentSystem implements PIDSource {
+	public static enum AlignmentState {
+		IDLE, ACTIVE;
+	}
+	protected final FusedSensor<Double> sensor;
+	protected final FusibleNavX navX;
+	protected AlignmentState state;
+	protected PIDSourceType pidSourceType;
+
+	public AlignmentSystem(AligningCamera camera, FusibleNavX navX) {
+		sensor = new FusedSensor<Double>(camera, navX);
+		this.navX = navX;
+		state = AlignmentState.IDLE;
+	}
+
+	public void start() {
+		state = AlignmentState.ACTIVE;
+		navX.zeroRelativeOffset();
+	}
+
+	public void cancel() {
+		state = AlignmentState.IDLE;
+	}
+
+	public double getDegrees() {
+		try {
+			return sensor.getValueDangerously();
+		}
+		catch (InvalidSensorException ex) {
+			LogKitten.wtf("AlignmentSystem: no sensors were available.");
```

Comment:
WTF is only really used for debugging. No sensors being available sucks, but it is not even `FATAL` and may not even be an `ERROR` (we might want it as a `WARN` although I think error is probably better)